src/Servers/Kestrel/test/InMemory.FunctionalTests/Properties/AssemblyInfo.cs

```diff
@@ -1,8 +1,10 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using Microsoft.AspNetCore.Testing.xunit;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Testing;
 
 [assembly: ShortClassName]
 [assembly: LogLevel(LogLevel.Trace)]
+[assembly: OSSkipCondition(OperatingSystems.MacOSX, SkipReason = "https://github.com/aspnet/AspNetCore/issues/8164")]
```

Comment:
@Tratcher @JunTaoLuo I copied this from https://github.com/aspnet/AspNetCore/blob/e6d77a1f6c1c87425e7820e6ded1ce527a66050d/src/Servers/HttpSys/test/FunctionalTests/Properties/AssemblyInfo.cs, but tests still run when I run `dotnet test` on my mac. Am I missing something?

Reply:
https://github.com/aspnet/Extensions/blob/master/src/TestingUtils/Microsoft.AspNetCore.Testing/src/xunit/OSSkipConditionAttribute.cs#L19

The skip reason is being interpreted as the os version probably. Turn the skip reason as a comment.

Reply:
It would be weird for a named parameter to be misinterpreted like that. Either way, I tried it and it doesn't appear to work.

As an aside, can we currently skip iff running on macOS AND as part of a Helix test?

Reply:
Turns out I have to use ConditionalFact/Theory 😆